src/DecisionTree.jl

```diff
@@ -83,6 +83,31 @@ function print_tree(leaf::Leaf, depth=-1, indent=0; feature_names=nothing)
     println("$(leaf.majority) : $(ratio)")
 end
 
+"""
+       print_tree(tree::Node, depth=-1, indent=0; feature_names=nothing)
+
+Print a textual visualization of the given decision tree `tree`.
+In the example output below, the top node considers whether 
+"Feature 3" is above or below the threshold -28.156052806422238.
+If the value of "Feature 3" is below the threshold for some input to be classified, 
```

Comment:
To be even clearer (relevant for discrete features):

```suggestion
If the value of "Feature 3" is strictly below the threshold for some input to be classified, 
```